paper-dropdown-menu.html

```diff
@@ -366,7 +390,17 @@
         // template. The metrics will change depending on whether or not the
         // input has a floating label.
         return noLabelFloat ? -4 : 8;
-      }
+      },
+
+      /**
+       * Returns false if the element is required and does not have a selection,
+       * and true otherwise.
+       * @return {Boolean} true if `required` is false, or if `required` is true
+       * and the element has a valid selection.
+       */
+      _getValidity: function() {
+        return this.disabled || !this.required || (this.required && this.value);
+      },
```

Comment:
We don't do this kind of trailing comma by convention (not that it's bad or anything.. also, Justin deliberately does the opposite for Designer, so maybe it's fine).